### PR TITLE
refactor(onboard/container): improve container documentation

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Improve container documentation

GitHub issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/Foundations/Layout/Container/code.mdx [hunk 1]
```diff
@@ -29,4 +33,6 @@ If you want to use only the default paddings, but still want the container to be
 </main>
 ```
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Container/code.mdx:35]: sentence change for `fluid container` part:

Use the ml-container--fluid modifiers if you want to use the default paddings only but still want the container to be fluid at any viewport width.

File: src/docs/Foundations/Layout/Container/code.mdx [hunk 2]
```diff
@@ -29,4 +33,6 @@ If you want to use only the default paddings, but still want the container to be
...
 
+## Related documentation
+
 To see container in action, all examples of [the grid system documentation](/Foundations/Layout/Grid/code/) uses it.
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Container/code.mdx:38]: To see **the** in action …

File: src/docs/Foundations/Layout/Container/index.mdx [hunk 3]
```diff
@@ -8,24 +8,37 @@ status:
 
 > The container defines the main content width as well as the minimum negative space between the horizontal edges of the screen and the content.
 
+Mozaic allows you to use two types of containers:
+
+- The **default container**: it's a fixed container that has a maximum width from **L** breakpoint and above
+- The **fluid container**: which has a width equal to 100% regardless of the breaking points
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Container/index.mdx:14]: <s>breaking points</s> breakpoints

File: src/docs/Foundations/Layout/Container/index.mdx [hunk 4]
```diff
@@ -8,24 +8,37 @@ status:
...
+
 ## Default container
 
 The default container follows these rules :
```
Comment (nherwegh) [inline src/docs/Foundations/Layout/Container/index.mdx:18]: rules:

**Review by nherwegh — CHANGES_REQUESTED** [review]
great improvement! well done 🎉

File: src/docs/Foundations/Layout/Container/index.mdx (hunk 4 above)
Comment (tiloyi) [reply src/docs/Foundations/Layout/Container/index.mdx]: fixed

File: src/docs/Foundations/Layout/Container/index.mdx (hunk 3 above)
Comment (tiloyi) [reply src/docs/Foundations/Layout/Container/index.mdx]: fixed

File: src/docs/Foundations/Layout/Container/code.mdx (hunk 2 above)
Comment (tiloyi) [reply src/docs/Foundations/Layout/Container/code.mdx]: fixed

File: src/docs/Foundations/Layout/Container/code.mdx (hunk 1 above)
Comment (tiloyi) [reply src/docs/Foundations/Layout/Container/code.mdx:35]: fixed

**Review by nherwegh — APPROVED** [review]

**Pull request merged**